bark/models/dynamic/single_track_steering_rate.hpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) 2020 fortiss GmbH
+//
+// Authors: Julian Bernhard, Klemens Esterle, Patrick Hart and
+// Tobias Kessler
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#ifndef BARK_MODELS_DYNAMIC_SINGLE_TRACK_STEERING_RATE_HPP_
+#define BARK_MODELS_DYNAMIC_SINGLE_TRACK_STEERING_RATE_HPP_
+
+#include <algorithm>
+#include <memory>
+#include "bark/commons/transformation/frenet_state.hpp"
+#include "bark/models/dynamic/dynamic_model.hpp"
+
+namespace bark {
+namespace models {
+namespace dynamic {
+
+
+class SingleTrackSteeringRateModel : public DynamicModel {
+ public:
```

Comment:
please add some doxygen documentation here... it is not obvious why the model consists of...